corehq/apps/groups/views.py

```diff
@@ -133,7 +133,7 @@ def update_group_membership(request, domain, group_id):
     form = MultipleSelectionForm(request.POST)
     form.fields['selected_ids'].choices = [(id, 'throwaway') for id in CommCareUser.ids_by_domain(domain)]
     if form.is_valid():
-        group.users = form.cleaned_data['selected_ids']
+        group.set_users(form.cleaned_data['selected_ids'])
```

Comment:
i wonder if there's a similar thing that needs to change in bulk user upload? how carefully did you search for explicit references to `.users`?